src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -153,77 +169,156 @@ export default class SimidMapCreative extends BaseSimidCreative {
- * @private 
- */
-displayMap_(coordinates = new google.maps.LatLng(DEFAULT_MAP_LAT, DEFAULT_MAP_LNG)) {
-  this.map_ = new google.maps.Map(document.getElementById('map'), {
-    zoom: DEFAULT_ZOOM,
-    center: coordinates
-  });
-  new google.maps.Marker({
-    position: coordinates,
-    map: this.map_,
-    title: 'Current Position'
-  });
-  this.findNearby_(this.searchQuery_, coordinates);
-}
+   * Loads a map object that currently defaults to a hardcoded location.
+   * @param {!google.maps.LatLng=} coordinates The LatLng object of user's current location.
+   * @private 
+   */
+  displayMap_(coordinates = new google.maps.LatLng(DEFAULT_MAP_LAT, DEFAULT_MAP_LNG)) {
+    this.currentLocation_ = coordinates;
+    this.map_ = new google.maps.Map(document.getElementById('map'), {
+      zoom: DEFAULT_ZOOM,
+      center: coordinates
+    });
+    new google.maps.Marker({
+      position: coordinates,
+      map: this.map_,
+      title: 'Current Position'
+    });
+    this.findNearby_(this.searchQuery_, coordinates);
+  }
 
-/**
- * Searches for the closest corresponding businesses based off of the given search parameter,
- * and places pins on the map that represent the 4 closest locations.
- * @param {String} searchParameter A string with the business's name to use in the query.
- * @param {!google.maps.LatLng} coordinates The LatLng object of user's current location.
- * @private 
- */
-findNearby_(searchParameter, coordinates) {
-  const request = {
-    location: coordinates,
-    name: searchParameter,
-    openNow: true,
-    rankBy: google.maps.places.RankBy.DISTANCE
-  };
-  const service = new google.maps.places.PlacesService(this.map_);
-  service.nearbySearch(request, this.displayResults_.bind(this));
-}
+  /**
+   * Searches for the closest corresponding businesses based off of the given search parameter,
+   * and places pins on the map that represent the 4 closest locations.
+   * @param {String} searchParameter A string with the business's name to use in the query.
+   * @param {!google.maps.LatLng} coordinates The LatLng object of user's current location.
+   * @private 
+   */
+  findNearby_(searchParameter, coordinates) {
+    const request = {
+      location: coordinates,
+      name: searchParameter,
+      openNow: true,
+      rankBy: google.maps.places.RankBy.DISTANCE
+    };
+    const service = new google.maps.places.PlacesService(this.map_);
+    service.nearbySearch(request, this.displayResults_.bind(this));
+    this.createTravelChoices_();
+  }
 
-/**
- * Displays the closest business locations to a user's current location.
- * @param {!Object} results An array of Place Results from the search query.
- * @param {!google.maps.places.PlacesServiceStatus} status The status returned 
- *  by the PlacesService on the completion of its searches.
- * @private 
- */
-displayResults_(results, status) {
+  /**
+   * Displays the closest business locations to a user's current location.
```

Comment:
don't specify business because it might be a park etc..

Reply:
Good call, I changed it!